frontend/components/ticket/TicketEvent.vue

```diff
@@ -97,36 +100,38 @@
 </template>
 
 <script>
-import Avatar from "../elements/Avatar";
-import Chip from "../elements/chip/Chip";
-import {calendarDate} from "../../utils";
-import Attachment from "./Attachment";
+  import Avatar from "../elements/Avatar";
+  import Chip from "../elements/chip/Chip";
+  import {calendarDate} from "../../utils";
+  import Attachment from "./Attachment";
 
-export default {
-  name: "TicketEvent",
-  props: ["ticket_event"],
-  components: {
-    Attachment,
-    Avatar,
-    Chip,
-  },
-  data() {
-    return {
-      date: calendarDate,
-      status: {
-        PNDG: 'Pending',
-        ASGD: 'Assigned',
-        ANSD: 'Awaiting response',
-        CLSD: 'Closed'
+  export default {
+    name: "TicketEvent",
+    props: ["ticket_event"],
+    components: {
+      Attachment,
+      Avatar,
+      Chip,
+    },
+    data() {
+      return {
+        date: calendarDate,
+        status: {
+          PNDG: 'Pending',
+          ASGD: 'Assigned',
+          ANSD: 'Awaiting response',
+          CLSD: 'Closed'
+        }
       }
-    }
-  },
-  methods: {
-    full_name: function (user) {
-      return `${user.first_name} ${user.last_name}`
     },
+    methods: {
+      full_name: function (user) {
+        if (user) {
+          return `${user.first_name} ${user.last_name}`
+        }
```

Comment:
Er wordt niet gereturned als users leeg is, misschien een ternary if van maken met een lege string.